inst/shiny-server/server/upload.R

```diff
@@ -123,10 +134,30 @@ observeEvent(selected_folders(), {
   else
     format <<- format_detected   # set the global data format
 
+  if(format == NEVERGRAD){
+    session$sendCustomMessage(type = "manipulateMenuItem", message = list(action = "hide", tabName = "RT_ECDF"))
+    session$sendCustomMessage(type = "manipulateMenuItem", message = list(action = "hide", tabName = "ERT_convergence"))
```

Comment:
Very good! So this is the generic, centralized way to control those tabs?

Reply:
Yes, this seems to be the easiest way to control tab visibility from inside R